.github/workflows/bbw_build_container_template.yml

```diff
@@ -85,17 +92,23 @@ jobs:
             done
 
       - name: No wsrep on 32 bit platforms
-        if: contains(inputs.platforms, 'linux/386')
+        if: ${{ (contains(inputs.platforms, 'linux/386')) && env.MAIN_BRANCH == 'false' }}
```

Comment:
The extra `(` around `contains` seems excessive.

`${{  ... }}` is probably overused on `step.x.if`.
ref: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsif